tests/tensor/test_slinalg.py

```diff
@@ -32,6 +32,10 @@
 from tests import unittest_tools as utt
 
 
+ATOL = 1e-8 if config.floatX.endswith("64") else 1e-4
+RTOL = 1e-8 if config.floatX.endswith("64") else 1e-4
```

Comment:
I don't love these global ATOL/RTOL. Future developers may blindly use them thinking they are well thought defaults for any linalg operations?